src/installation/thirdparty.md

```diff
@@ -8,6 +8,38 @@ Those methods are not officially supported, if you encounter installation issues
 
 A [Powershell script](https://github.com/losslesspng/SetUpKomgaJava) to get up and running with Komga.
 
+## Scoop (Windows)
+
+Komga is available in [Scoop](https://github.com/ScoopInstaller/Scoop)'s [extras](https://github.com/ScoopInstaller/Extras) bucket.
+
+### Installation
+
+You need Scoop to use this installation method. Instruction to install Scoop can be found [here](https://github.com/ScoopInstaller/Scoop#installation).
+
+#### 1. (Skip if JDK is installed) Add java bucket
+Run `scoop add bucket java`.
+
+#### 2. (Skip if JDK is installed) Install JDK
+Run `scoop install java/temurin-lts-jdk`.
+
+#### 3. (Skip if extras bucket is added) Add extras bucket
```

Comment:
What happens when you run this and the bucket is already added? Same question for the Java bucket.

If it doesn't complain if already added, we could probably simplify the instructions by always running it?

Reply:
```
PS C:\Users\LazyGeniusMan> scoop bucket add java
WARN  The 'java' bucket already exists. To add this bucket again, first remove it by running 'scoop bucket rm java'.
```
It shows warning and do nothing.

Reply:
Maybe we could have:
- an optional step for installing java, listing both `scoop add bucket java` and `scoop install java/temurin-lts-jdk` in a single step
- the install step as `scoop add bucket extras` + `scoop install komga

WDYT?

Reply:
Sounds good.

Reply:
Changed. Anything else?